src/main/java/com/fibermc/essentialcommands/mixin/ServerPlayerEntityMixin.java

```diff
@@ -39,6 +37,9 @@ public abstract class ServerPlayerEntityMixin extends PlayerEntityMixin implemen
     @Shadow
     public abstract boolean isSpectator();
 
+    @Shadow
+    public abstract boolean damage(ServerWorld world, DamageSource source, float amount);
+
```

Comment:
What's this for?

Reply:
Doubt past Arno would have a better answer for this, but present Arno really doesn't have a clue, doesn't even seem to have any usages, consider it gone